resources-service/package.json

```diff
@@ -4,13 +4,27 @@
   "description": "",
   "main": "dist/index.js",
   "devDependencies": {
+    "@tech-matters/testing": "^1.0.0",
     "sequelize": "^6.28.0",
     "sequelize-cli": "^6.5.2",
     "umzug": "^3.2.1"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "test:service:migrate": "node ./db-migrate",
+    "docker:compose:test:up": "docker compose -f ../hrm-service/docker-compose-test.yml up -d",
```

Comment:
Not for this PR and mostly just a note while this is making me think about it so it is not just in my head. 

I already have some ideas around refactoring/improving the local/test db setup, but in some future iteration the entire local db spin-up/tear-down system should probably move to the top level since it is no longer package specific.

Reply:
That may change in the next PR, since we are effectively service testing the HRM & resources DBs with separate databases then, each only deploys its own schema to the docker DB.

But there's certainly scope to tidy things up, there's a good bit of duplication between the two builds